services/stackexchange/stackexchange-reputation.service.js

```diff
@@ -0,0 +1,82 @@
+'use strict'
+
+const BaseJsonService = require('../base-json')
+const { metric } = require('../../lib/text-formatters')
+const { floorCount: floorCountColor } = require('../../lib/color-formatters')
+const Joi = require('joi')
+
+const reputationSchema = Joi.object({
+  items: Joi.array()
+    .min(1)
+    .items(
+      Joi.object({
+        reputation: Joi.number()
+          .min(0)
+          .required(),
+      })
+    )
+    .required(),
+}).required()
+
+module.exports = class StackExchangeReputation extends BaseJsonService {
+  static get category() {
+    return 'rating'
+  }
+
+  static get route() {
+    return {
+      base: 'stackexchange/r',
+      pattern: ':stackexchangesite/:query',
+    }
+  }
+
+  static get defaultBadgeData() {
+    return { label: 'stackoverflow' }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'StackOverflow',
```

Comment:
It would be better to call these **Stack Exchange reputation** / **Stack Exchange questions**.